.github/workflows/ci.yaml

```diff
@@ -244,3 +244,53 @@ jobs:
 
             # open html after being built
             htmlalyzer -b
+    upgrade-db:
+        name: Upgrade-Database
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v3
+        -
+            uses: actions/setup-python@v4
+            with:
+                python-version: '3.x'
+
+        - name: Upgrade V2 to V3
+          run: |
+            pip install -e .
+
+            # test ungathered default name
+            if [ -f .github/log_upgrade.sqlite ] ; then
+              rm .github/log_upgrade.sqlite
+            fi
+            upgrade-db .github/log.sqlite
```

Comment:
Can you rename this file to `log_ungathered_v2.sqlite`

Reply:
Ensure all combination of gathered/ungathered, v2/v3

Reply:
Is in CI